com.unity.render-pipelines.high-definition/Editor/Material/Decal/ShaderGraph/DecalMasterNode.cs

```diff
@@ -465,5 +465,13 @@ public ToggleData dotsInstancing
                 Dirty(ModificationScope.Graph);
             }
         }
+
+        public bool virtualTexturingEnabled
```

Comment:
Could write this more succinctly as:    public bool virtualTexturingEnabled => false;